src/app/zap-templates/zcl/data-model/chip/general-commissioning-cluster.xml

```diff
@@ -50,6 +63,26 @@ limitations under the License.
     <attribute side="server" code="0x02" define="REGULATORY_CONFIG" type="RegulatoryLocationTypeEnum" writable="false" optional="false">RegulatoryConfig</attribute>
     <attribute side="server" code="0x03" define="LOCATION_CAPABILITY" type="RegulatoryLocationTypeEnum" writable="false" optional="false">LocationCapability</attribute>
     <attribute side="server" code="0x04" define="SUPPORTS_CONCURRENT_CONNECTION" type="boolean" writable="false" default="1" optional="false">SupportsConcurrentConnection</attribute>
+    <attribute side="server" code="0x05" define="TC_ACCEPTED_VERSION" type="int16u" min="0x0000" max="0xFFFF" writable="false" optional="true" isNullable="true">
+      <description>TCAcceptedVersion</description>
+      <access op="read" privilege="view"/>
+      <access op="read" privilege="administer"/>
+    </attribute>
+    <attribute side="server" code="0x06" define="TC_MIN_REQUIRED_VERSION" type="int16u" min="0x0000" max="0xFFFF" writable="false" optional="true" isNullable="true">
+      <description>TCMinRequiredVersion</description>
+      <access op="read" privilege="view"/>
+      <access op="read" privilege="administer"/>
```

Comment:
These clash for every attribute. There cannot be 2 privileges for op == READ. I am surprised our tooling didn't catch this. Can you open an issue? 

For each of these, make them `<access op="read" privilege="administer"/>` (accurate by spec) and remove `<access op="read" privilege="view"/>`.

Reply:
Created [issue](https://github.com/project-chip/connectedhomeip/issues/34601), and updated the access attribute here.